### ISSUE TITLE
Training Issues: Very high training/validation loss

### ISSUE THREAD

**Issue by josch14** [salesforce/densecap#49]
First of all: Thanks for publishing the implemenatation code!

After many small errors occuring and small changes for fixing them, I am finally able to run the training of the model. However, my models (tested for different batch sizes, learning rates and for both, the Masked Transformer and the End-to-End Masked Transformer) won't really converge in a satisfying way. In one issue it was reported that 20 epochs should be enough for training. In contrast to that even after ~100 epochs I only get into the range of minimum 2,0 for the training/validation loss.

Did anyone else experience this? Do you have an idea what the reason for this could be? 

![Unbenannt](https://user-images.githubusercontent.com/86748031/127015956-800ca655-364d-4444-b392-b197919a1835.PNG)

**Comment by josch14** [comment]
I can't really understand why, but setting up the project up again with slightly different/older packages made it work now. 

For the masked transformer and batch size of 32 the first training/validation loss now is 2.3234/1.67. (train loss > val loss !?).
